Sources/Liquid/Variable.swift

```diff
@@ -36,8 +36,24 @@ struct Variable {
 	
 	private func parseFilterArgs(_ parser: Parser) -> [Expression] {
```

Comment:
Lets change our return type here from `[Expression]` to a tuple along the lines of `([Expression], [String: Expression])`